### PR TITLE
Fix/tao 4556 result celldata

### PR DESCRIPTION
**Pull request by ssipasseuth**
This PR fixes the output filterCellData() when the filter is set to "all" results.

### CODE REVIEW

File: scripts/update/Updater.php [hunk 1]
```diff
@@ -57,9 +57,9 @@ public function update($initialVersion)
             $this->setVersion('2.6.1');
         }
 
-        $this->skip('2.6.1', '4.3.0');
```
Comment (siwane) [inline scripts/update/Updater.php]: Why this change?

Comment (ssipasseuth) [inline scripts/update/Updater.php]: because of this tag https://github.com/oat-sa/extension-tao-outcomeui/releases/tag/v4.3.1.
you think we should release another hotfix instead 4.3.2 ?

Comment (siwane) [inline scripts/update/Updater.php]: this update script is not a part of tag? It does not change anything in this PR you can leave it here

Comment (ssipasseuth) [inline scripts/update/Updater.php]: I think v4.3.1 failed to be forward ported into develop, that is why. Now the question is does the client needs 4.3.1 ?
